specification/web/resource-manager/Microsoft.Web/stable/2022-09-01/StaticSites.json

```diff
@@ -1529,39 +2255,39 @@
           }
         },
         "x-ms-examples": {
-          "List custom domains for a static site": {
-            "$ref": "./examples/GetStaticSiteCustomDomains.json"
+          "List overviews of database connections for the static site.": {
+            "$ref": "./examples/GetStaticSiteDatabaseConnections.json"
           }
         },
         "x-ms-pageable": {
           "nextLinkName": "nextLink"
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/staticSites/{name}/customDomains/{domainName}": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/staticSites/{name}/databaseConnections/{databaseConnectionName}": {
       "get": {
         "tags": [
           "StaticSites"
         ],
-        "summary": "Gets an existing custom domain for a particular static site.",
-        "description": "Description for Gets an existing custom domain for a particular static site.",
-        "operationId": "StaticSites_GetStaticSiteCustomDomain",
+        "summary": "Returns overview of a database connection for a static site by name",
+        "operationId": "StaticSites_GetDatabaseConnection",
         "parameters": [
           {
             "$ref": "#/parameters/resourceGroupNameParameter"
           },
           {
             "name": "name",
             "in": "path",
-            "description": "Name of the static site resource to search in.",
+            "description": "Name of the static site",
             "required": true,
             "type": "string"
           },
           {
-            "name": "domainName",
+            "name": "databaseConnectionName",
             "in": "path",
-            "description": "The custom domain name.",
+            "description": "Name of the database connection that should be retrieved",
             "required": true,
+            "pattern": "^[a-zA-Z0-9]+$",
             "type": "string"
           },
```

Comment:
You might want to refactor this, to reduce copy/paste risk - put this in "Parameters" block and reference it everywhere with $ref.